# -*- coding: utf-8 -*-
from django.test import TestCase


class Issue45ErrorLoadingOauthStoreModule(TestCase):
    def test_store_import(self):
        from oauth_provider.store import store
        self.foo(store)

    def test_import_user_from_compat(self):
        from oauth_provider.compat import get_user_model
        from oauth_provider.compat import AUTH_USER_MODEL

        self.foo(foo())
        self.foo(AUTH_USER_MODEL)
